spring-transaction/src/main/kotlin/org/jetbrains/exposed/spring/SpringTransactionManager.kt

```diff
@@ -212,7 +290,15 @@ class SpringTransactionManager(
         override fun isRollbackOnly() = isRollback
 
         override fun flush() {
-            // Do noting
+            TransactionSynchronizationUtils.triggerFlush()
```

Comment:
`DataSourceTransactionManager.DataSourceTransactionObject` (which is the transaction object) had this on `flush`. I don't fully understand why it's there, yet, sadly.

Reply:
Reading up on it more closely - it looks like it was added to `DataSourceTransactionManager` to be compatible with a scenario where you use it combined with things like JPA but without using the `JpaTransactionManager`.

This makes it feel a bit niche to me - and I decided to restore the "Do nothing" case that was there before

https://github.com/spring-projects/spring-framework/issues/19413